### PR TITLE
Let the my-area cards wrap again

### LINKED ISSUE #139: My Area cards don't wrap
Probably has something to do with the new flex version

image

### PR DESCRIPTION
**Pull request by TimvdLippe**
Fixes #139

### CODE REVIEW

File: app/elements/lancie-my-area/lancie-my-area.html [hunk 1]
```diff
@@ -27,25 +27,25 @@
...
-      <my-area-teams class="flex-2 card"></my-area-teams>
+    <div class="layout horizontal start content wrap">
+      <my-area-profile user="{{user}}" class="flex card"></my-area-profile>
+      <my-area-teams class="flex card"></my-area-teams>
     </div>
-    <div class="layout horizontal start content">
+    <div class="layout horizontal start content wrap">
       <my-area-orders class="flex card"></my-area-orders>
-      <my-area-seat class="card"></my-area-seat>
+      <my-area-seat class="flex card"></my-area-seat>
```
Comment (skamoen) [inline app/elements/lancie-my-area/lancie-my-area.html]: Remove this `flex` for now, it will be changed in #61

**Comment by TimvdLippe** [comment]
Feedback required from @martijn9612 or @bvanderdrift 

Is the current spacing/sizing good? The spacing will change a little bit with the materialize orders branch.

**Comment by bvanderdrift** [comment]
We'll take a look into this this afternoon

**Comment by TimvdLippe** [comment]
LGTM

**Pull request merged**